### PR TITLE
Implement Add and Remove

### PR DESCRIPTION
**Pull request by edenlisk**
In this PR I did the following:
1. Implement Add.
2. Implement Remove.
3. Implement Update.

### CODE REVIEW

File: src/todolist/script.js [hunk 1]
```diff
@@ -1,57 +1,122 @@
...
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.classList.add('checkbox');
+    if (this.toDoInfo[index].completed === true) {
+      listDescription.classList.add('text-decoration-line-through');
+      checkbox.checked = true;
+    } else {
+      listDescription.classList.add('text-decoration-none');
+      checkbox.checked = false;
+    }
```
Comment (m05e5) [inline src/todolist/script.js:37]: - [ ] Hi please make sure to save the status(completed) to local storage too

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @edenlisk , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/todolist/script.js (hunk 1 above)
Comment (edenlisk) [reply src/todolist/script.js:37]: Hi @m05e5,
I have got your point but I guess we are supposed to implement that feature in third milestone.

Comment (GzimAsani) [reply src/todolist/script.js:37]: - [ ] This is the third milestone, in this one you need to implement the status being saved on the local storage as well. On the project requirements, it is specified that all the changes must be saved on local storage. The next milestone is for testing so kindly update the required change here
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @edenlisk                               :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your project, it creates and deletes items :+1: 

 ## Required Changes 
- [ ] The millstone that you submitted your project is the third one, kindly check the images and implement the changes asked by the previous reviewer
image
image
image




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @edenlisk, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights ✨✨
- No Linters errors ✅
- Followed GitHub flow ✅
- Great working on the functionality 🔥🚀🎉
- Descriptive PR ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**